spec/lib/client_spec.rb

```diff
@@ -18,7 +18,7 @@
       stub = stub_login_request({with_body: body})
       stub.to_return(:status => 200, :body => fixture("login_response")) #, :headers => {})
 
-      subject.login(username: 'testing', password: 'password_and_token')
+      subject.login(username: 'testing', password: 'password_and_token', filters: [])
```

Comment:
Adding filters here doesn't look right. It's not used by the `login` method, it's only used from `initialize`.